config/runtime.exs

```diff
@@ -148,7 +148,7 @@ case Keyword.get(args, :log_file) do
     config :logger, :default_formatter,
       format: {LogfmtEx, :format},
       colors: [enabled: false],
-      metadata: [:mfa]
+      metadata: [:mfa, :slot]
```

Comment:
do we want `:root` here too?

Reply:
Yes, but I'm leaving it for another PR. Reason: just one of the two gives enough context, and `:root` is given as a raw binary that's processed by the other formatter, however the default one can't.